.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,28 @@
+## What does this PR do?
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)